_scripts/l10n/clone.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
+
+# Script for cloning l10n repo into a shared location. This script takes no arguments,
+# but rather relies on env variables for configuration. All variables are optional.
+# Here's what they do:
+# - FXA_L10N_REPO - The target l10n git hub repo that contains translations
+# - FXA_L10N_BRANCH - The target branch of the repo. Defaults to default branch.
+# - FXA_L10N_DEPTH - The number of recent commits to fetch. Defaults 100.
```

Comment:
Worth noting (just here in review) that this used to be 1000. I'm guessing that's due to the huge number of small commits that come from Pontoon, but 100 seems fine to me personally.

Reply:
Yeah I did this on purpose. 1000 seemed like a lot, let's try it out.